CONTRIBUTING.md

```diff
@@ -68,3 +81,28 @@ git add -A
 git commit -sm "one commit on <branch-name>"
 git push --force
 ```
+
+## Code Quality
+
+This project is using [pre-commits](https://pre-commit.com) to ensure the quality of the code. Every change
```

Comment:
Let's emphasize it's not a must

```suggestion
This project is using [pre-commits](https://pre-commit.com) to ensure the quality of the code, however, this is not a must to contribute.

  Every change
```

Reply:
How about:
> We encourage you to use pre-commits but it's not a must. 

Imho same meaning but has a nice call to action. Believe me, once you start using pre-commits you you will appreciate them 😄

Reply:
I believe you and I use it myself, but we should not force people to do so.

I'll leave it up to @zroubalik to decide.

Reply:
Yeah. I think "Encourage" is a good word.

In Apache Airflow this often works like that:

a) contributor submit CI build several times and it fails for different reason
b) the contributor fixes it manually and re-submits (and it fails again as it was not perfect fix)
c) one of the committers gently suggests "we encourage you to use pre-commits" to save your time (and points out to the dos) 
d) the contributor comes back with green PR with "I wish I knew pre-commit before" comment ...
e) the contributor starts advising other contributors to use pre-commit

Reply:
Thanks for the suggestion.

Reply:
Yeah I'd definitely add a note over there as @tomkerkhove suggested. We shouldn't force that or gave users such impression. I'd add a note something like this `We encourage you to use pre-commits but it's not a must to contribute.`

Reply:
Added